tfx/orchestration/kubeflow/e2e_tests/kubeflow_gcp_perf_test.py

```diff
@@ -241,7 +246,7 @@ def testFullTaxiGcpPipeline(self):
         'parameterServerCount': self._PARAMETER_SERVER_COUNT
     }
 
-    pipeline = penguin_kubeflow_gcp.create_pipeline(
+    pipeline = _penguin_kubeflow_gcp.create_pipeline(
```

Comment:
It seems like this should fail because `tfx.examples.penguin._penguin_kubeflow_gcp` is empty. Should this instead be referring to `tfx.examples.penguin.penguin_pipeline_kubeflow.create_pipeline`?

Reply:
Yes. Fixed.